src/BlockTypes/ProductQuery.php

```diff
@@ -120,9 +120,11 @@ public function update_query( $pre_render, $parsed_block ) {
 	 * @param WP_REST_Request $request Request.
 	 */
 	public function update_rest_query( $args, $request ) {
-		$on_sale_query = $request->get_param( '__woocommerceOnSale' ) !== 'true' ? array() : $this->get_on_sale_products_query();
-		$orderby_query = $this->get_custom_orderby_query( $request->get_param( 'orderby' ) );
-		$tax_query     = $this->get_product_attributes_query( $request->get_param( '__woocommerceAttributes' ) );
+		$orderby        = $request->get_param( 'orderby' );
+		$woo_attributes = $request->get_param( '__woocommerceAttributes' );
+		$on_sale_query  = $request->get_param( '__woocommerceOnSale' ) === 'true' ? $this->get_on_sale_products_query() : array();
+		$orderby_query  = isset( $orderby ) ? $this->get_custom_orderby_query( $orderby ) : array();
+		$tax_query      = isset( $woo_attributes ) ? $this->get_product_attributes_query( $woo_attributes ) : array();
```

Comment:
I mean, just because the return value of `get_product_attributes_query` returns a `tax_query`, so I thought it would be a bit more explicit on what's going on when glancing at the code.

Reply:
Good point, it makes sense to rename it `attributes_query`.